template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Class-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common"
+  ],
+  "name": "Class",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Creates a new C# class",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "DotNet.Common.Class",
```

Comment:
"defaultName" should be set to `Class1`, once https://github.com/dotnet/templating/pull/5744 is merged it should work.

Reply:
done